lib/ansible/module_utils/facts.py

```diff
@@ -937,6 +937,18 @@ def get_dmi_facts(self):
     @timeout(10)
     def get_mount_facts(self):
         self.facts['mounts'] = []
+        bind_mounts = []
+        findmntPath = module.get_bin_path("findmnt")
+        rc, out, err = module.run_command("%s -lnur" % ( findmntPath ), use_unsafe_shell=True)
```

Comment:
i would make this all conditional to findmnt being found or it will error out on systems that don't have it